products/ceqr_survey/models/product/ceqr_survey_bbls.sql

```diff
@@ -30,13 +30,11 @@ ceqr_flags_wide AS (
         {% for row in ceqr_variables -%}
             MAX(
                 CASE
-                    WHEN
-                        variable_type = '{{ row["variable_type"] }}'
-                        THEN variable_id
+                    WHEN variable_type = '{{ row["variable_type"] }}' THEN variable_id
```

Comment:
Curious if you tried dbt-utils.pivot for this?

Reply:
Ooh. I did not!

Reply:
There are also some neat macros around things like [deduplication](https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#deduplicate-source), which could potentially obviate the window fn. Something to play with later!